actions/push-to-gar-docker/README.md

```diff
@@ -71,6 +71,12 @@ input.
 | `labels`                  | List    | List of custom labels to add to the image as metadata (see the [metadata-action](https://github.com/docker/metadata-action?tab=readme-ov-file#inputs)) for details             |
 | `target`                  | String  | Name of the [build stage](https://docs.docker.com/build/building/multi-stage/) to target.                                                                                      |
 | `delete_credentials_file` | Boolean | Delete the credentials file after the action is finished. If you want to keep the credentials file for a later step, set this to false. (Default: `true`)                      |
+| `secrets`        | List    | Secret(s) to expose to the build. Only needed when authenticating to private repositories outside the repository in which the image is being built.                            |
+
+```yaml
+secrets: |
+  GIT_AUTH_TOKEN=${{ secrets.MYTOKEN }}
+```
```

Comment:
I noticed you added an example under the table, but no other inputs have an example here, I think this section can be removed, it looks like a [lost piece of YAML](https://github.com/grafana/shared-workflows/blob/3743fad8cafc740a09c3a784d5eb9658e2dc4270/actions/push-to-gar-docker/README.md#outputs) in the middle of the readme.